doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -0,0 +1,806 @@
+
+Device          Name                    Cause                       Time                                User    Comment
+
+switch          2023_10_20_18_52_28     Watchdog:1 expired;         Wed 20 Oct 2023 06:52:28 PM UTC     N/A     N/A
+switch          2023_10_05_18_23_46     reboot                      Wed 05 Oct 2023 06:23:46 PM UTC     user    N/A
+DPU0            2023_10_04_18_23_46     Power Loss                  Tue 04 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3            2023_10_03_18_23_46     Watchdog: stage 1 expired;  Mon 03 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3            2023_10_02_18_23_46     Host Power-cycle            Sun 02 Oct 2023 06:23:46 PM UTC     N/A     Host lost DPU
+DPU3            2023_10_02_17_23_46     Host Reset DPU              Sun 02 Oct 2023 05:23:46 PM UTC     N/A     N/A
+DPU2            2023_10_02_17_20_46     reboot                      Sun 02 Oct 2023 05:20:46 PM UTC     admin   User issued 'reboot'
+
+"show reboot-cause history <module-name>"
+
+root@sonic:~#show reboot-cause history dpu3
+
+Device      Name                    Cause                           Time                                User    Comment 
+   
+DPU3        2023_10_03_18_23_46     Watchdog: stage 1 expired;      Mon 03 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3        2023_10_02_18_23_46     Host Power-cycle                Sun 02 Oct 2023 06:23:46 PM UTC     N/A     Host lost DPU
+DPU3        2023_10_02_17_23_46     Host Reset DPU                  Sun 02 Oct 2023 05:23:46 PM UTC     N/A     N/A
+```
+
+show chassis modules status - shows the dpu status of all DPUs and the Switch supervisor     <font>**`Executed on the switch`**</font></p>
+```
+root@sonic:~#show chassis modules status                                                                                      
+Name        Description         Physical-Slot       Oper-Status     Admin-Status    Serial
+
+DPU0        SS-DPU-0            1                   Online          up              SN20240105
+SWITCH      Chassis             0                   Online          N/A             FLM27000ER
+```
+* The system health summary on NPU should include the DPU status. If one or more DPUs are not ok it should be highlighted in the command output
+* This will allow to reuse of the existing infrastructure that allows to signal the user about the issue on the system and change the system status LED to red.
+
+show system-health summary      <font>**`Executed on the switch`**</font></p>
+```
+System status summary
+
+  System status LED  red
+  Services:
+    Status: Not OK
+    Not Running: dpu:dpu0, dpu:dpu1
+  Hardware:
+    Status: OK
+ ```
+ * Detailed output from the switch can be obtained with the following CLI
+ * The system health monitor list command should include the status of the DPU. If one or more DPUs are not ok it should be highlighted in the command output
+ * The switch will fetch the DPU information stored in the DPU's STATE_DB as needed
```

Comment:
let us specify how (what transport) is used get DPU health

Reply:
A gRPC client running on the switch PMON will fetch the data.  I'll updating the document.